dbt/adapters/databricks/impl.py

```diff
@@ -256,47 +255,67 @@ def get_relations_without_caching(self, relation: DatabricksRelation) -> Table:
 
         # if there are any table types to be resolved
         if any(not row[3] for row in new_rows):
-            # Get view names and create a dictionay of view name to materialization
+            # Get view names and tables and create a dictionary of name to materialization
+            relation_all_tables = self.Relation.create(
```

Comment:
Create a relation named in such a way as to pass the wildcard into the `like` clause.